data/com/mojang/math/SymmetricGroup3.mapping

```diff
@@ -0,0 +1,9 @@
+CLASS com/mojang/math/SymmetricGroup3
+	METHOD <init> (Ljava/lang/String;IIII)V
```

Comment:
This method signature does not exist.

Reply:
This is possibly an implication with enumerations specifically, where the string and integer represent the name and the ordinal of the enum value.